commitment/commitment_test.go

```diff
@@ -13,6 +14,39 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func randKey(t *testing.T) *btcec.PrivateKey {
+	t.Helper()
+	key, err := btcec.NewPrivateKey()
+	require.NoError(t, err)
+	return key
+}
+
+func randGenesis(t *testing.T, assetType asset.Type) asset.Genesis {
+	t.Helper()
+	return asset.Genesis{
+		FirstPrevOut: wire.OutPoint{},
+		Tag:          "",
+		Metadata:     nil,
+		OutputIndex:  rand.Uint32(),
+		Type:         assetType,
+	}
+}
+
+func randGroupKey(t *testing.T, genesis asset.Genesis) *asset.GroupKey {
```

Comment:
Looks like this was added (and in a few other places), but never used. I think we can also just consolidate to the new internal testing package.

Reply:
I think there were already, was a bad rebase on #211.

Reply:
There's the `asset.RandGroupKey()` and `asset.RandGenesis()` that can be used here instead.